pygmt/src/meca.py

```diff
@@ -305,91 +300,84 @@ def meca(  # noqa: PLR0912, PLR0913, PLR0915
         The component of the seismic moment tensor to plot.
 
         - ``"full"``: the full seismic moment tensor
-        - ``"dc"``: the closest double couple defined from the moment tensor
-          (zero trace and zero determinant)
+        - ``"dc"``: the closest double couple defined from the moment tensor (zero
+          trace and zero determinant)
         - ``"deviatoric"``: deviatoric part of the moment tensor (zero trace)
     longitude : float, list, or 1-D numpy array
-        Longitude(s) of event location(s). Must be the same length as the
-        number of events. Will override the ``longitude`` values
-        in ``spec`` if ``spec`` is a dictionary or :class:`pandas.DataFrame`.
+        Longitude(s) of the event location(s). Length must match the number of events.
+        Overrides the ``longitude`` values in ``spec`` if ``spec`` is a dictionary or
+        :class:`pandas.DataFrame`.
     latitude : float, list, or 1-D numpy array
-        Latitude(s) of event location(s). Must be the same length as the
-        number of events. Will override the ``latitude`` values
-        in ``spec`` if ``spec`` is a dictionary or :class:`pandas.DataFrame`.
+        Latitude(s) of the event location(s). Length must match the number of events.
+        Overrides the ``latitude`` values in ``spec`` if ``spec`` is a dictionary or
+        :class:`pandas.DataFrame`.
     depth : float, list, or 1-D numpy array
-        Depth(s) of event location(s) in kilometers. Must be the same length
-        as the number of events. Will override the ``depth`` values in ``spec``
-        if ``spec`` is a dictionary or :class:`pandas.DataFrame`.
+        Depth(s) of the event location(s) in kilometers. Length must match the number
+        of events. Overrides the ``depth`` values in ``spec`` if  ``spec`` is a
+        dictionary or :class:`pandas.DataFrame`.
```

Comment:
Combine docstrings of the three parameters into one entry? E.g.,
```
    longitude/latitude/depth : float, list, or 1-D numpy array
        Longitude(s)/latitude(s)/depth(s) of the events. Length must match the number
        of events. Overrides the ``longitude``/``latitude``/``depth`` values in 
        ``spec`` if  ``spec`` is a dictionary or :class:`pandas.DataFrame`.
```

Reply:
Combined the docstring for `longitude`, `latitude`, and `depth` as well as for `plot_longitude` and `plot_latitude` into one entry, respectively.